packages/oid4vci-demo-frontend/src/configs/fmdm.json

```diff
@@ -172,7 +172,7 @@
   "routes": [
     {
       "id": "default",
-      "vpDefinitionId": "sphereonGuest",
+      "vpDefinitionId": "fma",
```

Comment:
not sure about this change. I've recently changed this to sphereon and the reason is that it doesn't make sense to ask for a credential that we're issuing

Reply:
This should match the definition id that is asking for a Sphereon Wallet Identity credential.

Reply:
There is only a pd requesting a GuestCredential,  I created a new pd for Sphereon Wallet Identity credential.